### PR TITLE
Hopefully solved the issue with the missing emails

### PR DESCRIPTION
**Pull request by Hammarn**
I had previously added the email recipients to the wrong call of `transfer_run` this one is the correct one that's being called automatically by TACA after demultiplexing.

### CODE REVIEW

File: taca/analysis/analysis.py [hunk 1]
```diff
@@ -89,6 +89,9 @@ def _upload_to_statusdb(run):
     """ Triggers the upload to statusdb using the dependency flowcell_parser
         :param Run run: the object run
     """
+    return
+    import pdb
```
Comment (vezzi) [inline taca/analysis/analysis.py]: debug code

**Comment by vezzi** [comment]
Yep I see what the problem was. Remove the debugging code and I will merge and deploy. Well done

File: taca/analysis/analysis.py (hunk 1 above)
Comment (Hammarn) [inline taca/analysis/analysis.py]: removed

**Comment by vezzi** [comment]
perfect!!!

**Comment by vezzi** [comment]
@Hammarn I cannot merge it, I previously merged senthil OR and now your version conflicts. So you need to bump the version again.

**Comment by Hammarn** [comment]
Version increased!

**Pull request merged**